Microsoft.Toolkit.Uwp.UI/Extensions/SurfaceDialTextbox/SurfaceDialTextbox.cs

```diff
@@ -58,15 +58,15 @@ public class SurfaceDialTextbox
             DependencyProperty.RegisterAttached("EnableTapToNextControl", typeof(bool), typeof(SurfaceDialTextbox), new PropertyMetadata(true));
 
         /// <summary>
-        /// EnableMinMax limits the value in the textbox to your spesificed Min and Max values, see the other properties.
+        /// EnableMinMax limits the value in the textbox to your specified Min and Max values, see the other properties.
         /// </summary>
         public static readonly DependencyProperty EnableMinMaxValueProperty =
             DependencyProperty.RegisterAttached("EnableMinMaxValue", typeof(bool), typeof(SurfaceDialTextbox), new PropertyMetadata(false));
 
         /// <summary>
         /// Getter of the EnableMinMax property
         /// </summary>
-        /// <param name="obj">The Depenency Object we are dealing with, like a TextBox.</param>
+        /// <param name="obj">The Dependency Object we are dealing with, like a TextBox.</param>
```

Comment:
Side note, ideally all these type references should be proper XML refs, for consistency and also to allow devs to just CTRL + click into any of these to read more about them through the VS jump to definition feature.